docker_enforcer.py

```diff
@@ -132,6 +132,22 @@ def show_recent_stats():
     return show_filtered_stats(lambda c: c.last_timestamp > docker_helper.last_check_containers_run_start_timestamp)
 
 
+@app.route('/recent_detections_with_all_violated_rules')
```

Comment:
other URLs don't inlcude "detections". Can you rename to something like `recent_with_violated_rules`?